source/linux/basic/wildcards.rst

```diff
@@ -0,0 +1,40 @@
+Wildcards
+=========
+
+Wildcards are simple but powerful "tools". They are more the less autocomplete for paths or filenames. You can use wildcards to simplify big tasks like removing all files that have a filename with a specific extension. 
+Wildcard Characters:
+
+\* (Asterisk)
+^^^^^^^^^^^^^
+    Description
+        The Shell replaces the asterisk with everything, that is available.
+        In the case of the example the shell would replace the \*.o with a list of all files that fit the wildcard. 
+        But only in the working directory. The asterisk does not include subdirectories.
+    
+    Usage
+        For Example deleting all files with .o extension
+        .. code-block:: bash
+            
+            $host rm \*.o
+        
```

Comment:
You could also describe ** for subdirs:
like `ll **/*.o` for listing all .o files in all subdirs.